incubator/hnc/test/e2e/demo_test.go

```diff
@@ -61,4 +63,25 @@ var _ = Describe("Demo", func() {
 		RunShouldContain(nsTeamB+"-wizard", 5, "kubectl -n", nsService1, "get roles")
 		RunShouldNotContain(nsTeamA+"-wizard", 5, "kubectl -n", nsService1, "get roles")
 	})
+
+	It("Should propagate different types", func(){
+		MustRun("kubectl create ns", nsOrg)
+		MustRun("kubectl hns create", nsTeamA, "-n", nsOrg)
+		MustRun("kubectl hns create", nsTeamB, "-n", nsOrg)
+		MustRun("kubectl create ns", nsService1)
+		MustRun("kubectl hns set", nsService1, "--parent", nsTeamB)
+		MustRun("kubectl -n", nsTeamB, "create secret generic my-creds --from-literal=password=iamteamb")
+		// wait 2 seconds to give time for secret to propogate if it was to
+		time.Sleep(2*time.Second)
+		// secret does not show up in service-1 because we haven’t configured HNC to propagate secrets in HNCConfiguration.
+		RunShouldNotContain("my-creds", 2, "kubectl -n", nsService1, "get secrets")
+		MustRun("kubectl hns config set-type --apiVersion v1 --kind Secret propagate")
+		MustRun("kubectl get hncconfiguration config -oyaml")
```

Comment:
Seems we don't need this in the e2e test. This is for human to read and see the setting. Or you can have `RunShouldNotContain` secret before `k hns config set-type` line 78 and have `RunShouldContain` secret here.

Reply:
If a human was supposed to see it, then we should check that it's there in the demo. Just be careful because the spec and status look very similar, and we should definitely be looking for the status here. I _think_ there's a way to just get the status from an object using kubectl, maybe look into that.

Reply:
No we don't need to have this cmd here. I put it here because this is part of the demo, and I want to confirm that the user can run this cmd without any error. I'm not sure if it's necessary tho. Should I remove it instead?